SSM/SSM.java

```diff
@@ -191,5 +194,14 @@ public void onMobTarget(EntityTargetEvent e) {
         e.setCancelled(true);
     }
 
+    public static void teleportAllPlayers() {

```

Comment:
Please just put this in the game manager itself

Reply:
yeah this was supposed to be in game manager but when i added it i just put it in ssm and havent changed it since.....